pkg/radius/handler/handler.go

```diff
@@ -121,44 +121,122 @@ func HandleRadiusAccessRequest(udpConn *net.UDPConn, tngfAddr, ueAddr *net.UDPAd
 
 	case EAP5GNAS:
 		radiusLog.Infoln("Handle EAP-Res/5G-NAS")
-		var eap *radius_message.EAP
+		eap := &radius_message.EAP{}
 		err = eap.Unmarshal(eapMessage)
 		if err != nil {
 			radiusLog.Errorf("[EAP] EAP5GNAS unmarshal error: %+v", err)
```

Comment:
We should return if unmarshal error occurs!

Reply:
Okay. Thank you.